tests/common/util.rs

```diff
@@ -3872,4 +3894,21 @@ mod tests {
             .no_stderr()
             .stdout_is("8\n16\n");
     }
+
+    #[cfg(unix)]
+    #[test]
+    fn test_altering_umask() {
+        use uucore::mode::get_umask;
+        let p_umask = get_umask();
+        //make sure we are not testing against the same umask
```

Comment:
```suggestion
        // make sure we are not testing against the same umask
```
Just a tiny suggestion :)